packages/create-sitecore-jss/src/init-runner.ts

```diff
@@ -41,16 +41,19 @@ export const initRunner = async (initializers: string[], args: BaseArgs) => {
 
   await runner(initializers);
 
-  saveConfiguration(args.templates, path.resolve(`${args.destination}${sep}package.json`));
-  // final steps (install, lint, etc)
-  if (!args.noInstall) {
-    installPackages(args.destination, args.silent);
-    lintFix(args.destination, args.silent);
-  }
+  for (const destination of [args.destination, args.nodeAppDestination]) {
+    if (!destination) continue;
+    saveConfiguration(args.templates, path.resolve(`${destination}${sep}package.json`));
+    // final steps (install, lint, etc)
+    if (!args.noInstall) {
+      installPackages(destination, args.silent);
+      lintFix(destination, args.silent);
```

Comment:
Our proxy apps don't have linting configured. Skip this for those?

Reply:
Do we intend it to not have lint, ever?

Reply:
I think it's better to call this and other util functions regardless and let the create logic check whether the commands are present or not in package.json. This is simpler, more intuitive and allows us worry less if we decide to add lint and other things to proxies later on.
Prior to this PR the functions would be launched anyway when create-sitecore-jss is called for proxy only.

Reply:
Ah, you're right. I agree, makes sense to leave it then.